src/mbgl/style/style_impl.cpp

```diff
@@ -36,6 +36,16 @@ Style::Impl::Impl(FileSource& fileSource_, float pixelRatio)
     light->setObserver(this);
 }
 
+Style::Impl::Impl(FileSource& fileSource_, float pixelRatio, CameraOptions camera)
+    : Impl(fileSource_, pixelRatio) {
```

Comment:
why not to std::move camera into defaultCamera?

Reply:
Done